apps/web/src/views/Swap/V3Swap/components/SwapModalFooter.tsx

```diff
@@ -226,6 +244,37 @@ export const SwapModalFooter = memo(function SwapModalFooter({
             </Text>
           )}
         </RowBetween>
+        {isPaymasterAvailable && isPaymasterTokenActive && (
+          <RowBetween mt="8px">
+            <RowFixed>
+              <Text fontSize="14px">{t('Gas Token')}</Text>
+            </RowFixed>
+
+            <Flex alignItems="center">
+              <Text marginRight={2} fontSize={14}>
+                {(feeToken && feeToken.symbol && feeToken.symbol.length > 10
+                  ? `${feeToken.symbol.slice(0, 4)}...${feeToken.symbol.slice(
+                      feeToken.symbol.length - 5,
+                      feeToken.symbol.length,
+                    )}`
+                  : feeToken?.symbol) || 'ETH'}
+              </Text>
+
+              <div style={{ position: 'relative' }}>
+                <StyledLogo
```

Comment:
There's a `CurrencyLogo` component in internal-widgets package

Reply:
I had tried the local CurrencyLogo component but it didn't produce most images. The one from internal-widgets works 👍🏻

Reply:
yea we should remove the one in web app